docs/fbc_onboarding.md

```diff
@@ -0,0 +1,113 @@
+# File Based Catalog onboarding
+
+Operators in certified, marketplace, or community repositories are defined in a declarative way.
+This means a user provides all necessary information in advance about the operator bundle and how it
+should be released in a catalog and OPM automation injects a bundle into the correct place
+in the upgrade path.
+
+This is however very limited solution that doesn't allow any further modification of upgrade
+paths after a bundle is already released. Due to this limitation, a concept
+of FBC (File-based catalog) is now available and allows users to modify the operator upgrade
+path in a separate step without the need to release a new bundle.
+
+To enable FBC for a given operator the operator owner needs to convert
+existing operator into FBC format.
+
+We want to help with this process and we prepared a tooling that helps with this transition.
+
+## Convert existing operator to FBC
+As a prerequisite to this process, you need to install a few dependencies.
+
+```bash
+# Go to the operator repo directory (certified-operators, marketplace-operators, community-operators-prod)
+cd certified-operators
+
+# Install a migration script
+pip install git+https://github.com/redhat-openshift-ecosystem/operator-pipelines.git
+
+# Download opm cli tool
+curl -sL https://github.com/operator-framework/operator-registry/releases/download/v1.39.0/linux-amd64-opm -o opm && \
```

Comment:
I would consider to adding at least a link to the official install documentation https://docs.openshift.com/container-platform/4.15/cli_reference/opm/cli-opm-install.html
Btw, version 1.40 was released today.